packages/sdk/CHANGELOG.md

```diff
@@ -2,9 +2,12 @@
 
 ### Unreleased
 
+### 2.2.0-rc.0
+
 - (major) refactor: change message status to a tagged union instead of an enum,
   added the root to the `proven` message status
 - fix: checkHomes async handling in for loop
+- fix: import `fetch` from cross-fetch
```

Comment:
don't add a new dep. use axios, which we already depend on

Reply:
Discussed in DMs.  For visibility, because of issues with cors headers and disabling caching, sticking with cross-fetch and removing axios